src/content/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring.mdx

```diff
@@ -200,6 +200,10 @@ When requesting a CSP exception, follow your organization's standard procedures
 
 The browser JavaScript agent requires the following CSP exceptions in order to run properly. Alternatively, you can add all of these exceptions to the fallback `default-src` directive (instead of `script-src` and `connect-src`).
 
+<Callout variant="important">
+In older browsers `image-src` or requests for images can be a possible exception that should be accounted for.
+</Callout>
+
```

Comment:
I think @austin-schaefer would have more context on the classification intended by older browsers.

Reply:
This was originally filed thru DOC-6340. We think your edit will work just fine, esp. since we haven't heard any more about this potential problem since then. Thanks so much for fixing this doc!